### PR TITLE
describe changes in ROI Shape transforms

### PR DESCRIPTION
**Pull request by mtbc**
This new blog entry addresses https://trello.com/c/9YhVKx4U/449-roi-transforms by warning of the breaking changes caused by https://github.com/openmicroscopy/openmicroscopy/pull/4672.

Albeit with broken formatting one can get a view of the text at https://github.com/mtbc/blog/blob/transforms/_posts/2016-06-15-shape-transforms.md. For proper formatting use Jekyll locally.

### CODE REVIEW

**Comment by dominikl** [comment]
Looks alright to me! 👍

**Comment by mtbc** [comment]
Thank you @dominikl! Much appreciated, as you also reviewed the corresponding PR. After @sbesson and @hflynn have taken a look I'll copy over to gdoc for @jrswedlow but fine if all that's *after* PotN.

File: _posts/2016-06-15-shape-transforms.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+<script type="text/javascript"
+  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=MML_HTMLorMML">
+</script>
+
+The OME Data Model defines a
+[Shape.Transform](https://www.openmicroscopy.org/Schemas/Documentation/Generated/OME-2015-01/ROI_xsd.html#Shape_Transform)
+element with an
+[AffineTransform](https://www.openmicroscopy.org/Schemas/Documentation/Generated/OME-2015-01/ROI_xsd.html#AffineTransform)
+type that is used to transform a Shape's position in the image in
+various ways: scaling, rotations and more. In order to properly locate a
```
Comment (sbesson) [inline _posts/2016-06-15-shape-transforms.md]: Worth including 2D to define the limit of the transformations?

**Comment by sbesson** [comment]
Couple of thoughts post review:
- generally on the review process
   - I needed https://github.com/ome/blog/pull/31 to build this page locally using Jekyll 3.1.6. The pagination gem is explicitly included as a dependency on the GH pages side (see https://github.com/github/pages-gem/pull/175). Is it worth reincluding for our own development/review workflow or are there workarounds?
   - I also needed to rename this page to prevent using a future date - I assume it's a blog feature
- while we mention Java client upgrade, do we have/need to collect more information on OMERO.web?
- once cleared, I think some of the information in this blog post could also be copied over to the ome.xsd  hence the generated Oxygen documentation for the schema and the OMERO developer documentation

**Comment by hflynn** [comment]
Now I've got this building locally using the same method as Seb, it's looking good thanks! I second the suggestion that we add a link to the developer docs once we publish this. I leave you to decide what action is appropriate re: OMERO.web but from my side, good to be cleared by Jason and then published with an appropriate date change.

**Comment by hflynn** [comment]
P.S. have merged the pagination gem addition so everyone with the latest version of jekyll will be able to build locally once they fetch the update.

**Comment by mtbc** [comment]
OMERO.web: from my impression of the Python and JavaScript code I'm also fairly optimistic about it: I think it's already using matrices of a corresponding form. I just mentioned the Java as an example to show that the speculation is founded!

**Comment by mtbc** [comment]
I hadn't realized the local staging issues, my apologies for that trouble. To allow people to avoid Jekyll issues entirely I've staged the blog myself: for this PR see http://users.openmicroscopy.org.uk/~mtbcarroll/vpn/blog/data-model/future-plans/2016/06/15/shape-transforms/. @sbesson: You'll need to VPN to see it.

**Comment by hflynn** [comment]
@mtbc when you get chance, can you update the filename date to 2016-06-20 and then I'll get this out next week

**Comment by mtbc** [comment]
Will do so later today.

**Pull request merged**